frontend/src/DeviceDetailsPage/DeviceDetailsConciseRow.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import Proptypes from "prop-types";
+import { TableContainer, TableCell, TableRow } from "@material-ui/core";
+import ChannelDetailsTable from "../devicelist/ChannelDetailsTable";
+import InputChannelInfo from "../model/InputChannelInfo";
+import OutputChannelInfo from "../model/OutputChannelInfo";
+
+export default class DeviceDetailsConciseRow extends React.Component {
+  static getPropertyDisplayName(name) {
+    switch (name) {
+      case "serialNumber":
+        return "Serial Number";
+      case "lastCommunication":
+        return "Last Communication";
+      case "ip":
+        return "IP Address";
+      case "name":
+        return "Name";
+      case "status":
+        return "Status";
+      case "channels":
+        return "Channels";
+      default:
+        return "Additional Info";
+    }
+  }
+
+  static createInnerTable(value) {
+    return (
+      <TableContainer>
+        {value.map((channel) => {
+          return (
+            <ChannelDetailsTable
+              channel={channel}
+              key={`chDetails_${channel.id}`}
+            />
+          );
+        })}
+      </TableContainer>
+    );
+  }
+
+  render() {
+    const { name, value } = this.props;
+    return (
+      <>
```

Comment:
i've never seen `<>` on its own, what does this do? :O

Reply:
It's linter's version of `<React.Fragment/>`

Reply:
!? ok, interesting choice for the linter xD
thanks for the explanation~